README.md

```diff
@@ -1,35 +1,55 @@
 # HOOMD-blue benchmarks
 
 This repository contains a number of performance benchmarks for HOOMD-blue. These are run on a variety of hardware
-and execution configurations and the results are stored in the repository. Jupyter notebooks summarize the results
-and the rendered notebook is checked in to the repository so that it can be viewed at nbviewer.org. Links to these
-benchmark results are posted on https://glotzerlab.engin.umich.edu/hoomd-blue/benchmarks.html
+and execution configurations and the results are stored as [signac](http://signac.io) job documents.
+
+## Prerequistes
+
+* [signac and signac-flow (groups branch)](http://signac.io)
+* [HOOMD-blue (next branch)](http://glotzerlab.engin.umich.edu/hoomd-blue)
 
 ## Running benchmarks standalone
 
-`cd` into a benchmark directory and run `python bmark.py`. It will print out performance information
-and overwrite a `metadata.json` file with the performance numbers and execution configuration metadata. Each benchmark
-should run in serial and MPI, though some may be too big for a single GPU's memory.
+To submit all benchmarks as cluster jobs,
+
+```
+python project.py submit -w 2 --exec
+```
 
+or, to run interactively, extract the command lines of the individual operations
 ```
-cd microsphere
-python bmark.py
-mpirun -n 4 python bmark.py
+python project.py submit --exec --test
```

Comment:
`--exec` effectively reverts to previous behavior. There is a bug in the master branch of `signac` that will point to where ever operations are defined. For this to work since operations are defined in different files from where `main` is called, the groups branch is needed. However, due to not properly handling mpi currently, `--exec` is necessary to handle mpi tests.